src/containers/home.js

```diff
@@ -1,9 +1,23 @@
 import React from 'react';
 import styled from 'styled-components';
 import { device } from '../device';
+import NextSection from '../components/nextSection';
+
+const HomeContainer = styled.div`
+    background: #FCF7EC;
+    width: 1440px;
+    height: 1950px;
+`
```

Comment:
```suggestion
const HomeContainer = styled.main`

`;
```
I added the background color to the global styles so you can take it off here. Also, don't hard-code a width and height of the main page wrapper. You never know the size of a screen and its better to just let it fit the screen rather than force it